lib/ansible/modules/include_vars.py

```diff
@@ -56,6 +56,12 @@
     type: list
     elements: str
     version_added: "2.2"
+  follow_symlinks:
```

Comment:
As requested [here](https://github.com/ansible/ansible/issues/79922#issuecomment-1423222158), we would like to follow symlinks without adding a new option.

This will also need tests and a [changelog](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-changelog-fragments).